Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM debian:jessie
+MAINTAINER Praekelt Foundation <dev@praekeltfoundation.org>
+
+RUN apt-get -qq update && apt-get -qqy install python-pip python-dev
+RUN pip install junebug
+RUN mkdir logs
+
+EXPOSE 8080
+ENTRYPOINT ["jb", "--config=/app/example-config.yaml"]
```

Comment:
How do people view/keep their logs if they aren't on a volume though? Maybe
Junebug should create the logging folder if it doesn't exist? Not sure.